proxy/src/main/java/com/velocitypowered/proxy/VelocityServer.java

```diff
@@ -172,6 +191,9 @@ void start() {
     commandManager.register("shutdown", new ShutdownCommand(this),"end");
     commandManager.register("glist", new GlistCommand(this));
 
+    loadTranslations("en_us");
```

Comment:
Should we hardcode every lang here? Listing and loading all the resources in the JAR `translations` directory might be a better option.


```java
final String translationsPath = "translations/";
final File jarFile = new File(getClass().getProtectionDomain().getCodeSource().getLocation().getPath());

try (final JarFile jar = new JarFile(jarFile)) {
final Enumeration<JarEntry> entries = jar.entries();

while (entries.hasMoreElements()) {
        final String name = entries.nextElement().getName();
        if (name.startsWith(path)) {
            // TODO load translation
        }
    }
}
```

(sorry for formatting, I'm on mobile)

Reply:
Yeah, this is on my TODO list.